fe/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -762,4 +762,13 @@ public void reset() {
         dataPartition = null;
         targetColumns.clear();
     }
+
+    @Override
+    public RedirectStatus getRedirectStatus() {
+        if (isExplain()) {
+            return null;
+        } else {
+            return RedirectStatus.FORWARD_WITH_SYNC;
```

Comment:
I think `FORWARD_NO_SYNC` is enough. Explain Insert does not modify any meta data that need to be sync.

Reply:
`RedirectStatus.FORWARD_WITH_SYNC` is for insert, not  Explain Insert. So We should use `FORWARD_WITH_SYNC`

Reply:
O, i see